app.go

```diff
@@ -308,11 +337,51 @@ func New(opts ...Option) *App {
 
 	if err := app.executeInvokes(); err != nil {
 		app.err = err
+		var b bytes.Buffer
+
+		if dig.CanVisualizeError(err) {
+			dig.Visualize(app.container, &b, dig.VisualizeError(err))
+		}
+
+		errorHandlerList(app.errorHooks).onError(errorWithGraph{
+			graph: b.String(),
+			err:   err,
```

Comment:
We probably want to call `onError` with the original error if no wrapping was
done instead of wrapping with an empty buffer.

Let's change this to,

```
if dig.CanVisualizeError(err) {
    var b bytes.Buffer
    dig.Visualize(..)
    err = errorWithGraph{...}
}

errorHandlerList(app.errorHooks).onError(err)
```